### ISSUE TITLE
CEF4 Delphi - with Thinfinity Virtual UI

### ISSUE THREAD

**Issue by NDzubiel** [salvadordf/CEF4Delphi#391]
Hello @all,

I am trying to embed Chrome into my application - that I am using with Thinfinty Virtual UI -
but I am getting this error - I guess it has to do something with the browser display/draw functions -
I tried several things - but nothing seems to work ... has anybody already solved
this problem ?

Thanks.

Nils

Faulting application name: SimpleBrowser.exe, version: 1.0.0.0, time stamp: 0x617e68be
Faulting module name: libcef.dll, version: 95.7.14.0, time stamp: 0x6179e9ee
Exception code: 0x80000003
Fault offset: 0x02647633
Faulting process id: 0x994
Faulting application start time: 0x01d7ce464df00d37
Faulting application path: C:\Google\SimpleBrowser.exe
Faulting module path: C:\Google\libcef.dll
Report Id: 30dff0dd-71d0-4cf8-a7ca-46852f8e9340
Faulting package full name: 
Faulting package-relative application ID:

**Comment by salvadordf** [comment]
I tried the Thinfinity VirtualUI demos available at their website using the MiniBrowser demo and all of them work correctly.

SimpleBrowser and many other demos block new popup windows to keep them as simple as possible but the MiniBrowser demo can handle them. Perhaps VistualUI needs popup windows to work.

**Comment by salvadordf** [comment]
I haven't tested that yet but I wouldn't be surprised if the SimpleOSRBrowser demo works fine when you virtualize it with Thinfinity Virtual UI.
Could you try that demo too?

**Comment by salvadordf** [comment]
I installed the trial and tried to run a few demos and all of them crash inside Thinfinity Virtual UI.
Even the official CEF sample application can't start in Thinfinity Virtual UI.

**Comment by salvadordf** [comment]
Not only the GPU subprocesses crash. Also the Network service is crashing.
It seems that all Chromium subprocesses crash when you run them inside Thinfinity Virtual UI.

I tried setting GlobalCEFApp.SingleProcess to true and the browser starts correctly but this is an unsupported mode and it has shutdown problems.
I tried running the demo with the credential of another user with admin rights and the problem is the same.
I also tried all these settings without success : 
  GlobalCEFApp.EnableGPU := False;
  GlobalCEFApp.AddCustomCommandLine('--disable-gpu-process-crash-limit');
  GlobalCEFApp.AddCustomCommandLine('--disable-gpu-rasterization');
  GlobalCEFApp.AddCustomCommandLine('--disable-gpu-vsync');
  GlobalCEFApp.AddCustomCommandLine('--disable-gpu-watchdog');
  GlobalCEFApp.AddCustomCommandLine('--disable-software-rasterizer');

Perhaps Thinfinity Virtual UI doesn't allow to launch any other process from a hosted app.
